lzero/policy/efficientzero.py

```diff
@@ -473,6 +475,8 @@ def _forward_learn(self, data: torch.Tensor) -> Dict[str, Union[float, int]]:
         total_grad_norm_before_clip = torch.nn.utils.clip_grad_norm_(
             self._learn_model.parameters(), self._cfg.grad_clip_value
         )
+        if self._cfg.multi_gpu:
```

Comment:
You should sync gradient first, then do grad clip

Reply:
ok